phpunit/class-gutenberg-rest-navigation-fallback-controller-test.php

```diff
@@ -158,10 +158,39 @@ public function test_prepare_item() {
 	}
 
 	/**
-	 * @doesNotPerformAssertions
+	 * Tests that the correct filters are applied to the context parameter.
+	 *
+	 * The REST response for the Posts Controller doesn't return all fields,
+	 * when the context is set to embed. We need to add additional fields
+	 * for the navigation fallback, so that when it embeds a navigation,
+	 * the required fields are present.
+	 *
+	 * @covers wp_add_fields_to_navigation_fallback_embeded_links
 	 */
 	public function test_context_param() {
```

Comment:
```suggestion
	public function test_embedded_navigation_post_contains_required_fields() {
```

Let's make this explicit.